### PR TITLE
fix(dev-env/basic): building and running development

### PR DESCRIPTION
**Pull request by renatobenks**
### Description

The development environment, aka `dev-env/basic`, is not properly working, currently because of a build problem.

### Review

- [ ] Changes on script `dev` in `dev-env/basic`
- [ ] Lock of dependencies on `yarn`, just to make sure that I'm not upgrading anything that I shouldn't 

### Pre-merge checklist

- [ ] Check `dev` script on `dev-env/basic` which should be working now

### Screenshots

| Before | After |
| ------ | ----- |
| image | Screen Shot 2021-07-29 at 18 10 17 |

### CODE REVIEW

File: dev-env/basic/package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "cpx": "^1.5.0",
-    "docz": "next",
+    "docz": "../../core/docz",
```
Comment (pedronauck) [inline dev-env/basic/package.json:18]: 💯

**Review by pedronauck — CHANGES_REQUESTED** [review]
This happened here when I tried to run `yarn dev` inside `dev-env/basic` and check `localhost:3000`

Screen Shot 2021-07-29 at 19 00 48

**Comment by renatobenks** [comment]
I'm checking it! Thank you for testing it

**Comment by renatobenks** [comment]
@pedronauck I just checked everything around by setting this up from scratch and it seems to be working fine now. Can you try it again, but this time from a clean setup?

Let me know how it works then.

**Pull request merged**